Samples/PackageWithExternalLocation/cs/PhotoStoreDemo/ExecutionMode.cs

```diff
@@ -42,7 +39,7 @@ internal static string GetSafeAppxLocalFolder()
         {
             try
             {
-               // return Windows.Storage.ApplicationData.Current.LocalFolder.Path;
+                // return Windows.Storage.ApplicationData.Current.LocalFolder.Path;
```

Comment:
This method is useless... we should either just delete it or uncomment out the useful line :)

Reply:
Removed.